docs/kb/integrations/wiz.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: wiz
+append_help_link: true
+title: View Semgrep findings in Wiz's Security Graph
+hide_title: true
+description: "Learn how to view Semgrep findings in Wiz's Security Graph."
+tags:
+ - Wiz
+ - Semgrep AppSec Platform
+---
+
+# View Semgrep findings in Wiz's Security Graph
+
+Semgrep integrates with Wiz by calling Wiz’s GraphQL API endpoints and uploading your static analysis (SAST) vulnerability findings to a dedicated Amazon S3 bucket. Your Semgrep SAST vulnerability findings are mapped to the same correlated repository scanned by Wiz and enriched by any available inventory and runtime-related data, such as clusters, pods, containers, cloud configurations, and more. Semgrep's goal is to give you a holistic view of your code and infrastructure security so that you can focus on what matters most.
+
+## Prerequisites and requirements
+
+This integration is available for users with both a [Semgrep Code license](https://semgrep.dev/products/semgrep-code/) and a [Wiz Code Security license](https://www.wiz.io/platform/wiz-code). 
+
+To send Semgrep Code findings to Wiz:
+
+- You must have a Wiz service account with sufficient permissions to create a service account, if needed, and integrations. The service account must be able to provide Semgrep with the following scopes: `create:external_data_ingestion`, `read:system_activities`, and `read:resources`. If you don't have a service account:
+    1. Create a [Wiz service account](https://docs.wiz.io/wiz-docs/docs/service-accounts-settings?lng=en). When prompted to select the **Type** of the service account, select **Custom Integration (GraphQL API)**.
+    2. Copy the Wiz **Client ID** and **Client Secret** provided. You must provide this information to Semgrep at a later stage.
+- You must add the [Semgrep integration](https://app.wiz.io/settings/automation/integrations) from the Wiz Integration Network. During this process, save the following values shown to you:
+   1. API Endpoint URL
+   2. Authentication URL
```

Comment:
They will be able to find API endpoint URL and authentication URL at this address: https://app.wiz.io/tenant-info/general